src/Service.Tests/Authorization/AuthorizationHelpers.cs

```diff
@@ -137,6 +129,25 @@ public static AuthorizationResolver InitAuthorizationResolver(RuntimeConfig runt
             return runtimeConfig;
         }
 
+        /// <summary>
+        /// Helper method to create runtime settings with given authentication provider.
+        /// </summary>
+        /// <param name="authProvider">Authentication provider.</param>
+        /// <returns>Created runtime settings.</returns>
+        public static Dictionary<GlobalSettingsType, object> CreateRuntimeSettings(string authProvider = "AppService")
```

Comment:
#1402 will see this revisited as we now have a type rather than a dictionary for these settings.